tox.ini

```diff
@@ -34,7 +34,7 @@ deps =
     black
     ruff
 commands =
-    ruff check --fix {[vars]all_path}
+    ruff format {[vars]all_path}
```

Comment:
I think `ruff check --fix` and `ruff format` are two different things, and we should probably run both; what do you think?

Reply:
Since this is the fmt environment, I think it's probably best to have it only fix formatting. Ruff didn't have this command before but now it does so I think this is better.

Reply:
Personally I don't want my format command to change the content of my code but if that's what we decided then fine